awx/main/analytics/collectors.py

```diff
@@ -311,7 +311,7 @@ def events_table(since, full_path, until, **kwargs):
     return _copy_table(table='events', query=events_query, path=full_path)
 
 
-@register('unified_jobs_table', '1.2', format='csv', description=_('Data on jobs run'), expensive=True)
```

Comment:
@Ladas have we actually shipped a version with 1.2 yet (does this need to actually change?)

Reply:
I am fine with both :-)

(we're not actually using the version for anything :-) , we will really need to do that only if we change e.g. the format of the field, in a way that would require conditional processing based on version)

Reply:
I'll change it back to `1.2`!